src/sentry/api/endpoints/organization_events.py

```diff
@@ -102,21 +100,19 @@ def get_legacy(self, request, organization):
             # or user doesn't have access to projects in org
             data_fn = lambda *args, **kwargs: []
         else:
-            snuba_cols = SnubaEvent.minimal_columns if full else SnubaEvent.selected_columns
+            cols = None if full else eventstore.full_columns
+
             data_fn = partial(
-                # extract 'data' from raw_query result
-                lambda *args, **kwargs: raw_query(*args, **kwargs)['data'],
-                selected_columns=snuba_cols,
-                orderby='-timestamp',
```

Comment:
same

Reply:
I think the default orderby (-timestamp, -event_id) is slightly better for consistent ordering in cases of same timestamp